src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
don't use `*` for this way
add margin: 0 to the body
you can use reset.css, google it